src/DqtApi/DataStore/Crm/DataverseAdapter.cs

```diff
@@ -312,6 +312,82 @@ private static void AddQualifiedTeacherStatusLink(QueryExpression query)
             qualifiedTeacherStatusLink.EntityAlias = nameof(dfeta_qtsregistration);
         }
 
+        public async Task<Account> GetOrganizationByProviderName(string providerName, params string[] columnNames)
+        {
+            var query = new QueryByAttribute(Account.EntityLogicalName)
+            {
+                ColumnSet = new ColumnSet(true)
```

Comment:
Use `columnNames` parameter here instead of `allColumns: true`

Reply:
done.